control-plane/api/v1alpha1/ingressgateway_types_test.go

```diff
@@ -453,6 +477,38 @@ func TestIngressGateway_Validate(t *testing.T) {
 		partitionEnabled  bool
 		expectedErrMsgs   []string
 	}{
+		"tls.minTLSVersion invalid": {
```

Comment:
Minor question: Should you test for at least 1 valid tlsMinVersion or tlsMaxVersion too?

Reply:
Great question. Updated the validate tests with valid config for max and min TLS